src/python/pants/backend/jvm/subsystems/jar_dependency_management.py

```diff
@@ -255,7 +256,7 @@ def __iter__(self):
   def __len__(self):
     return len(self._artifacts_to_versions)
 
-  def __nonzero__(self):
+  def __bool__(self):
```

Comment:
`from builtins import object` means that this will implement `__nonzero__` for us if Py2

Reply:
I found through porting another module that this automatic implementation of __nonzero__ only holds if the class inherits the backported object. If it inherits another class, then there will be no __nonzero__ implementation. 

I think it’s likely safe to assume this won’t be subclassesed by something other than object anytime soon, but I can remove this deletion if we prefer to use the more explicit implementation of both __bool__ and __nonzero__.